doc/whats-new.md

```diff
@@ -146,12 +146,22 @@ What's new
 
 ### Bug fixes
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+- BoutMesh options now settable in GUI (#63)\
+=======
+=======
```

Comment:
These changes are due to merges. They indicate what has changed, and are not intended to be committed.

Here is some documentation on that:
https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#_basic_merge_conflicts